system-test/storage.ts

```diff
@@ -3129,6 +3129,20 @@ describe('storage', () => {
       assert.strictEqual(files!.length, NEW_FILES.length);
     });
 
+    it('returns folders as prefixes when includeFoldersAsPrefixes is set', async () => {
+      const expected = [`${DIRECTORY_NAME}/`];
+      const [, , result] = await bucket.getFiles({
```

Comment:
This doesn't confirm a specific managed folder is in the list of prefixes; It might be the best we can do right now though because you'd need to make a one-off request to create a managed folder through Apiary Node.js client to verify functionality.

Reply:
By apiary you mean storage control?

Reply:
No, there's support in Storage v1 json apiary that has CreateManagedFolder RPC which would help with this test until Storage Control is available.

Reply:
Probably beyond the scope of this change to start pulling in other dependencies just for a single test case.

Reply:
When do we plan on adding the create functionality in this library?